src/lsp/cobol_ptree/common.ml

```diff
@@ -30,3 +30,203 @@ type 'a nel = 'a NEL.t
 [@@deriving ord]
 
 let pp_nel ppe = NEL.pp ~fsep:"@ " ~fopen:"" ~fclose:"" ppe
+
+
+(* order *)
+
+type 'a order =
+  | P1 of 'a
+  | P2 of 'a
+  | P3 of 'a
+  | P4 of 'a
+[@@deriving ord]
+
+let (~!) = function
```

Comment:
Are those operators useful?

Reply:
These are only useful if we choose to go with the `'a order` type to define order